Lib/test/test_asyncio/test_taskgroups.py

```diff
@@ -12,11 +13,6 @@
 
 from test.test_asyncio.utils import await_without_task
 
-# To prevent a warning "test altered the execution environment"
```

Comment:
I know this isn't needed as you have changed it to use asyncio.EventLoop but let's keep it for now, it is easy to make a mistake on altering policy and is a pain to fix in CI if it happens without this.